packages/server-wallet/src/protocols/__test__/direct-funding.test.ts

```diff
@@ -1,12 +1,18 @@
-import {channel, channelWithVars} from '../../models/__test__/fixtures/channel';
+import {channelWithVars} from '../../models/__test__/fixtures/channel';
 import {protocol} from '../direct-funding';
 
 const c = channelWithVars();
-
+const protocolState = {
+  ...c.protocolState,
+  minimalOutcome: {
+    type: 'SimpleAllocation' as 'SimpleAllocation',
+    allocationItems: [],
+    assetHolderAddress: '0x0',
+  },
+};
```

Comment:
I'd like to try and keep the test data clean. So, this can be created in a fixture file for `direct-funding-state`.